scripts/docker/build.cmd

```diff
@@ -40,7 +40,16 @@ cd %APP_HOME%
 
     mkdir out\docker\webservice
 
-    dotnet publish WebService      --configuration %CONFIGURATION% --output bin\Docker
+    :: DotNet 3.x:
+    :: If a relative path is specified when publishing a project, the output directory generated 
+    :: is relative to the current working directory, not to the project file location.
+
+    :: DotNet 2.x:
+    :: If a relative path is specified when publishing a project, the output directory generated 
+    :: is relative to the project file location, not to the current working directory.
```

Comment:
Similar to comment I left on the other PR: consider deduping redundant text to make this clearer (and more concise)